torch/utils/_benchmark/utils/valgrind_wrapper/timer_interface.py

```diff
@@ -0,0 +1,448 @@
+"""Intermediate layer between `Timer` and `valgrind`."""
+import collections
+import dataclasses
+import os
+import re
+import shutil
+import subprocess
+import sys
+import tempfile
+import textwrap
+from typing import DefaultDict, Dict, Optional, Tuple
+
+import torch
+
+
+@dataclasses.dataclass(repr=False, eq=False, frozen=True)
+class CallgrindStats(object):
+    stmt: str
+    setup: str
+    number: int
```

Comment:
number... of runs? 👍

Reply:
Changed to `number_per_run` to be consistent with `Timer`/`Measurement`